FirebaseMLModelDownloader/Apps/Sample/Podfile

```diff
@@ -0,0 +1,13 @@
+platform :ios, '10.0'
+use_frameworks!
+
+pod 'FirebaseMLModelDownloader', :git => 'https://github.com/firebase/firebase-ios-sdk.git', :branch => 'sample-app'
```

Comment:
Would we like to use a local SDK version instead? I assume it will be more convenient for testing.
```
pod 'FirebaseMLModelDownloader',  :path => '../../'
```

Reply:
+1 Definitely

Reply:
Updated, thank you! I didn't know I could do that.